src/command_modules/azure-cli-resource/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 2.1.3
 +++++
 * `provider operation show`: exception handling to exit with code 3 upon a missing resource for consistency.
+* `managedapp create --kind MarketPlace`: fixed bug where this command to create an instance of a Marketplace managed app would crash.
```

Comment:
Sounds winded, remove the "this command" part as it's implied.
Suggestion:
`fixed bug causing instance creation of a Marketplace managed app to crash.

Reply:
You're right. This is a more concise description. I've updated the description.